backend/seed_data/test_cases.yaml

```
@@ -5,43 +5,85 @@ test_cases:
     target_runtime: O_N
     test_cases:
       - input: "[0,1,2,3,0]"
-        output: "true"
+        outputs: 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)